examples/crm/src/dataGenerator/deals.ts

```diff
@@ -46,8 +30,8 @@ export const generateDeals = (db: Db): Deal[] => {
             name: lowercaseName[0].toUpperCase() + lowercaseName.slice(1),
             company_id: company.id,
             contact_ids: contacts.map(contact => contact.id),
-            type: random.arrayElement(type),
```

Comment:
Why did you remove the type?

Reply:
I didn't delete it, I just renamed it category, as this term is used on the user side and in the specifications.